### PR TITLE
Space no longer instantly kills you

### PR DESCRIPTION
**Pull request by BraveMole**
## About The Pull Request

Still hurt a bunch

## Why It's Good For The Game

Useless instakill creates way more issues than it is supposed to resolve

## Changelog
:cl:
balance: Space no longer instantly kills you
/:cl:

### CODE REVIEW

**Comment by EgorDinamit** [comment]
Think the insta-kill thing was also added to some other tiles, like sea.
If they are not a subtype of "space", could you do something about them as well?

**Comment by BraveMole** [comment]
> Think the insta-kill thing was also added to some other tiles, like sea.
> If they are not a subtype of "space", could you do something about them as well?

sea is space

File: code/datums/status_effects/debuffs.dm [hunk 1]
```diff
@@ -371,3 +371,13 @@
 	REMOVE_TRAIT(owner, TRAIT_MUTED, TRAIT_STATUS_EFFECT(id))
 	return ..()
 
+/datum/status_effect/spacefreeze
+	id = "spacefreeze"
+
+/datum/status_effect/spacefreeze/on_creation(mob/living/new_owner)
+	. = ..()
+	to_chat(new_owner, span_danger("The cold vacuum instantly freezes you, maybe this was a bad idea?"))
```
Comment (AnCopper) [inline code/datums/status_effects/debuffs.dm:379]: "Cold vacuum" is an oxymoron.

Comment (BraveMole) [reply code/datums/status_effects/debuffs.dm:379]: why would it be? space has a temperature, it happens to be cold

**Comment by MereWalker** [comment]
I just want to point out the reason why the tile become INSTA kill was due to the wraith being very broken at the time. I don't support this but id like to see how this pans out regardless.

File: code/datums/status_effects/debuffs.dm (hunk 1 above)
Comment (AnCopper) [reply code/datums/status_effects/debuffs.dm:379]: Space is nothing, it literally can't be cold, because it's nothing.

Comment (BraveMole) [reply code/datums/status_effects/debuffs.dm:379]: You can define temperature stricly out of radiations. And if you consider this temperature, space is cold

**Comment by bowlofcereal** [comment]
Didnt we make them instakill because people were just setting up defenses in space to hide out in during a hijack, or the wraith teleporting out into space a lot? I'm not really against this, just hoping previous issues that caused it to become instakill in the first place can either be resolved or prevented

**Comment by BraveMole** [comment]
> Didnt we make them instakill because people were just setting up defenses in space to hide out in during a hijack, or the wraith teleporting out into space a lot? I'm not really against this, just hoping previous issues that caused it to become instakill in the first place can either be resolved or prevented

The issues were people staying in space. Marines build fort in space, wraith teleporting out and marines not being able to do anything.

Everyone still takes 25 fire damage every sec in space, so these strats are dead as well

**Comment by Alexeivichi** [comment]
Pretty sure space tiles insta-killing was to act as a "No-play" area. Doesn't make sense to make them not akin to a kill brushed area, other than turning them into a "hazard" rather than... Well, "No play deathzone".

Time to build a space bar once more I guess.

**Comment by bowlofcereal** [comment]
> > Didnt we make them instakill because people were just setting up defenses in space to hide out in during a hijack, or the wraith teleporting out into space a lot? I'm not really against this, just hoping previous issues that caused it to become instakill in the first place can either be resolved or prevented
> 
> The issues were people staying in space. Marines build fort in space, wraith teleporting out and marines not being able to do anything.
> 
> Everyone still takes 25 fire damage every sec in space, so these strats are dead as well

Okay, that works then. Cheers.

**Review by TiviPlus — APPROVED** [review]

**Pull request merged**